platform-visualization/src/tileManager.js

```diff
@@ -301,6 +301,39 @@ function TileManager() {
     };
 
     /**
+     * Compare two strings representing qualities
+     * @param   {String} a The first string representing a quality
+     * @param   {String} b The second string representing a quality
+     * @returns {Number} A number higher than 0 if a > b, a number lower than 0 if a < b, 0 is both are equal
+     */
+    function compareQuality(a, b) {
+        var table = {
```

Comment:
It's wrong to create the `table` variable with the hard-coded qualities, if we added another quality level then we should come here and add it too. When the config_tile.json file is read, you could save the `readJson.qualities` in a (constant) class variable (_QUALITIES_ for example) and then being _elm_ a tile element and _actualQa_ the actual quality you could just do `var skip = QUALITIES[elm.minQuality] >  QUALITIES[actualQa];`